backend/src/Designer/Services/Implementation/PreviewService.cs

```diff
@@ -6,29 +6,27 @@
 using Altinn.Platform.Storage.Interface.Models;
 using Altinn.Studio.Designer.Infrastructure.GitRepository;
 using Altinn.Studio.Designer.Models;
+using Altinn.Studio.Designer.Models.App;
 using Altinn.Studio.Designer.Services.Interfaces;
+using Altinn.Studio.Designer.Services.Interfaces.Preview;
 using LibGit2Sharp;
 
 namespace Altinn.Studio.Designer.Services.Implementation;
 
 /// <summary>
 /// Service for handling a mocked instance object for preview mode
 /// </summary>
-public class PreviewService : IPreviewService
+/// <remarks>
+/// Constructor
+/// </remarks>
+/// <param name="altinnGitRepositoryFactory">IAltinnGitRepository</param>
+/// <param name="dataService"></param>
+public class PreviewService(IAltinnGitRepositoryFactory altinnGitRepositoryFactory, IDataService dataService) : IPreviewService
```

Comment:
`dataService` does not seem to be in use here
```suggestion
public class PreviewService(IAltinnGitRepositoryFactory altinnGitRepositoryFactory) : IPreviewService
```

Reply:
Nice! Removed